anesthetic/samples.py

```diff
@@ -43,32 +44,40 @@ class MCMCSamples(WeightedDataFrame):
         loglikelihoods of samples.
 
     tex: dict
-        mapping from coloumns to tex labels for plotting
+        mapping from columns to tex labels for plotting
 
     limits: dict
-        mapping from coloumns to prior limits
+        mapping from columns to prior limits
+
+    label: str
+        Legend label
 
     """
 
-    _metadata = WeightedDataFrame._metadata + ['tex', 'limits', 'u', 'root']
+    _metadata = WeightedDataFrame._metadata + ['tex', 'limits', 'u',
+                                               'root', 'label']
```

Comment:
Not really for this PR, but as a side remark: `'u'` is very cryptic. I've gone through the code once to figure out what it does, but I forgot again... Is there maybe a more descriptive name for that variable?

Similar for `w`. Although, not as bad as it often appears with the key `'weight'` (and at least to me it's somewhat familiar/expected)...

Reply:
OK, I've added an issue #44 for further discussion and to remind us to address this at some point